### DISCUSSION TITLE
disabled "ip6cp" , subscriber doesn't become "Active"

### DISCUSSION THREAD

**Discussion by vbhurangi** [rtbrick/bngblaster#181] [Q&A]
Hi,
I am trying to bring up pppoe subscriber with ipv4 traffic only. On disabling "ip6cp" the subscriber doesn't come up. It goes into "configured" state. I checked the logs at bng router it is failing at the ipcp negotiation stage. 

Any hint to resolve the issue.

I have following config at  bngblaster side:

{
    "interfaces": {
         "rx-threads": 5,
         "tx-threads": 5,
         "io-slots": 16384,
         "network": {
             "interface": "ens5f2",
             "address": "200.0.0.1/24",
             "gateway": "200.0.0.5"
         },
         "access": [
             {
                "interface": "vlan90-if",
                "type": "pppoe",
                "inner-vlan": 90,
                "stream-group-id": 1,
                "authentication-protocol": "PAP"
             }
          ]
    },
    "pppoe": {
         "reconnect": true,
         "discovery-timeout": 30,
         "discovery-retry": 10
    },
    "ppp": {
            "mru": 1492,
            "authentication": {
            "username":"vbhurangi",
            "password":"brasvmx",
            "timeout": 5,
            "retry": 30
            },
           "lcp": {
            "conf-request-timeout": 1,
            "conf-request-retry": 10,
            "keepalive-interval": 30,
            "keepalive-retry": 3
            },
           "ipcp": {
            "enable": true,
            "request-ip": true,
            "request-dns1": true,
            "request-dns2": true,
            "conf-request-timeout": 1,
            "conf-request-retry": 10
            },
           "ip6cp": {
           "enable":false
           }
     },
     "dhcpv6": {
            "enable": false
     },
    "session-traffic": {
            "ipv4-pps": 500,
            "ipv6-pps":0,
            "ipv6pd-pps": 0
     }
}

**Comment by vbhurangi** [comment]
It worked on changing the ipcp configuration to below:-

 "ipcp": {
            "enable": true,
            "request-ip": true,
            "request-dns1": false,
            "request-dns2": false,
            "conf-request-timeout": 1,
            "conf-request-retry": 10
        },

**Reply by GIC-de** [reply]
Yes, if you request DNS, blaster fails if DNS is not provided. This ensures that test passed only if desired state is reached.
